File: level2/python/exercise3_easy.py
# Дано ціле число A. Потрібно написати програму,
#  яка виведе наступне за A парне число.
# Формат вхідних даних.  Подається одне ціле число A.
# Формат вихідних даних. 
# Виводиться наступне парне ціле число після A.


# def next_even(a: int) -> int:

test_cases = [0, 1, 2, 7, 10, -1, -4]
expected_results = [2, 2, 4, 8, 12, 0, -2]

import time


def test_next_even():
    start_time = time.time()
    for i, test in enumerate(test_cases):
        result = next_even(test)
        assert result == expected_results[i], f"Test case {test} failed: expected {expected_results[i]}, got {result}"
    end_time = time.time()
    print(f"All Python tests passed! Execution time: {end_time - start_time:.6f} seconds")

test_next_even()

File: level2/javascript/exercise3_easy.js
// Дано ціле число A. Потрібно написати програму,
//  яка виведе наступне за A парне число.
// Формат вхідних даних.  Подається одне ціле число A.
// Формат вихідних даних. 
// Виводиться наступне парне ціле число після A.

function nextEven(a) {
    
}


const testCases = [0, 1, 2, 7, 10, -1, -4];
const expectedResults = [2, 2, 4, 8, 12, 0, -2];

function testNextEven() {
    const startTime = performance.now();
    testCases.forEach((test, index) => {
        const result = nextEven(test);
        console.assert(result === expectedResults[index], `Test case ${test} failed: expected ${expectedResults[index]}, got ${result}`);
    });
    const endTime = performance.now();
    console.log(`All JavaScript tests passed! Execution time: ${(endTime - startTime).toFixed(6)} milliseconds`);
}

testNextEven();
